src/main/java/capstone/mfslbackend/service/FantasyTeamService.java

```diff
@@ -0,0 +1,32 @@
+package capstone.mfslbackend.service;
+
+import capstone.mfslbackend.model.FantasyTeam;
+import capstone.mfslbackend.repository.FantasyTeamRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@Slf4j
+public class FantasyTeamService {
+
+    private final FantasyTeamRepository fantasyTeamRepository;
+    public FantasyTeamService(FantasyTeamRepository fantasyTeamRepository) {
+        this.fantasyTeamRepository = fantasyTeamRepository;
+    }
+    public Optional<FantasyTeam> getFantasyTeam(Long teamId) {
+        Optional<FantasyTeam> fantasyTeam = fantasyTeamRepository.findById(teamId);
+        if (fantasyTeam.isEmpty()) {
+            log.warn("could not find team with id {}", teamId);
+        }
+        return fantasyTeam;
+    }
+
+    public FantasyTeam createFantasyTeam(String teamName) {
```

Comment:
When we create a team, should we have if statements and such to check if the user has submitted the request with an empty name variable. If so, we can send a warning saying a name is required. This can be handled in frontend too, but usually it is done in the backend since the frontend is supposed to have as less logic as possible. @seb8stien1 What do you think?